xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -30,8 +30,8 @@
   <parent>XWiki.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1465822369000</date>
-  <contentUpdateDate>1465822369000</contentUpdateDate>
+  <date>1465895028000</date>
```

Comment:
I don't agree Sergiu. I also commit this usually because it makes sense to me that when we make a modification the date is that of the modification and not keep the old date. Now that said we had discussions about this in the past but I don't think that the XAR plugin is resetting any date ATM (would need to check).